ckanext/fairdatapoint/profiles.py

```diff
@@ -1,7 +1,7 @@
 from ckanext.dcat.profiles import EuropeanDCATAP2Profile
 from ckan.plugins import toolkit
 from ckan import model
-
+import json
```

Comment:
a suggestion: to replace it with `import simplejson as json`, it is a little bit slower but we may get use of it's JSONDecoder/Encoder and exceptions